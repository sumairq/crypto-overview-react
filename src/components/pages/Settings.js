import React from 'react';

const Settings = () => (
  <div>Settings under construction.....</div>
);

export default Settings;
