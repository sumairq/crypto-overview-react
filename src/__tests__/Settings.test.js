import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Settings from '../components/pages/Settings';
import store from '../redux/configureStore';

test('matches settings component snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Settings />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
