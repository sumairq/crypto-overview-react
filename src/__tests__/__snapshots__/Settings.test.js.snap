// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches settings component snapshot 1`] = `
<div>
  Settings under construction.....
</div>
`;
